src/logging.c

```diff
@@ -1288,7 +1298,6 @@ void rcutils_logging_console_output_handler(
     is_colorized = IS_STREAM_A_TTY(g_output_stream);
   }
 
-  char msg_buf[1024] = "";
```

Comment:
These changes shall be done in the original repo: https://github.com/ros2/rcutils